Documentation/ceph-common-issues.md

```diff
@@ -11,73 +11,86 @@ Many of these problem cases are hard to summarize down to a short phrase that ad
 If after trying the suggestions found on this page and the problem is not resolved, the Rook team is very happy to help you troubleshoot the issues in their Slack channel. Once you have [registered for the Rook Slack](https://slack.rook.io), proceed to the General channel to ask for assistance.
 
 ## Table of Contents
-- [Troubleshooting Techniques](#troubleshooting-techniques)
-- [Pod using Rook storage is not running](#pod-using-ceph-storage-is-not-running)
-- [Cluster failing to service requests](#cluster-failing-to-service-requests)
-- [Monitors are the only pods running](#monitors-are-the-only-pods-running)
-- [PVCs stay in pending state](#pvcs-stay-in-pending-state)
-- [OSD pods are failing to start](#osd-pods-are-failing-to-start)
-- [OSDs are not created on my devices](#osd-pods-are-not-created-on-my-devices)
-- [Node hangs after reboot](#node-hangs-after-reboot)
-- [Rook Agent modprobe exec format error](#rook-agent-modprobe-exec-format-error)
-- [Rook Agent rbd module missing error](#rook-agent-rbd-module-missing-error)
-- [Using multiple shared filesystem (CephFS) is attempted on a kernel version older than 4.7](#using-multiple-shared-filesystem-cephfs-is-attempted-on-a-kernel-version-older-than-47)
-- [Activate log to file for a particular Ceph daemon](#activate-log-to-file-for-a-particular-ceph-daemon)
-- [Flex storage class versus Ceph CSI storage class](#flex-storage-class-versus-ceph-csi-storage-class)
-
-# Troubleshooting Techniques
+
+* [Table of Contents](#table-of-contents)
```

Comment:
Can we just remove the Table of Contents link?

Reply:
A vscode plugin is needed to generate this?

Reply:
I have made the Table of contents into a bold text instead of a heading as in the other file's case.
When autogenerating the table of contents it is a bit hard to exclude it.

I have added information on the plugin and settings to the development flow doc, see https://github.com/rook/rook/blob/9b7b391a7261202a003214d85f50d938bdcfe5e8/Documentation/development-flow.md.